app/main/views/add_service.py

```diff
@@ -46,13 +46,9 @@ def _create_example_template(service_id):
 @user_is_gov_user
 def add_service():
     default_organisation_type = current_user.default_organisation_type
-    if default_organisation_type == 'nhs':
-        form = CreateNhsServiceForm()
-        default_organisation_type = None
-    else:
-        form = CreateServiceForm(
-            organisation_type=default_organisation_type
-        )
+    form = CreateServiceForm(
+        # organisation_type=default_organisation_type
```

Comment:
Should this be uncommented?

Reply:
Ah yes, I'll add a new default here. UK used a series of functions to determine a user's org type based on their email address, which I don't think would work well for us. Let me form an opinion about whether we should keep that additional code and get back to you.

Reply:
Looks like default org is used in a few more places, so I left those functions in and created a new card to trim it down later.